frontends/pytorch/e2e_testing/torchscript/basic.py

```diff
@@ -101,3 +101,22 @@ def forward(self, x):
 @register_test_case(module_factory=lambda: ReluModule())
 def ReluModule_basic(module, tu: TestUtils):
     module.forward(tu.rand(4, 2) - 0.5)
+
+class BatchNormModule(torch.nn.Module):
+    def __init__(self):
+        super().__init__()
+        self.bn2d = torch.nn.BatchNorm2d(2)
+        self.bn2d.eval()
+        self.bn2d.running_mean = torch.tensor([0.5, 0.4])
```

Comment:
can you remove running_mean/var for now and add a TODO explaining the work remaining to be done?

Reply:
If the `running_mean/var` is not given the input will always be the same as the output. The Batchnorm calculation would not really be fully tested.  Maybe I can add comments to explain why the fake data is added for testing purpose?

Reply:
Oh, I see, you are initializing the BatchNorm2d instance. For some reason I thought it was `self.running_mean`. Sorry!